### ISSUE TITLE
Any plan to migrate it to composer

### ISSUE THREAD

**Issue by dezudas** [nababur/Pro-User-Management-System-with-PHP-MySQL-Ajax#3]
Do we have any plan to move this to composer and further enhancement?

**Comment by nababur** [comment]
Yes, I have a plan to use composer.

**Comment by dezudas** [comment]
Thank you for make it open source, few things to make it better
- migrate to composer
- Fix issue with .sql file, fix default value issue
- fix registration page issue
- remove hardcoded email

**Comment by nababur** [comment]
Yes, you are right I don't get time to work again on this project. But in the future, I have a plan to release another new user management in git and YouTube video tutorial please subscribe to my Channel.
